torch/csrc/jit/fuser/cpu/temp_file.h

```diff
@@ -15,6 +25,97 @@ namespace jit {
 namespace fuser {
 namespace cpu {
 
+#ifdef _MSC_VER
+/* mkstemps extracted from gcc/libiberty/mkstemps.c.  Copyright (C) 1991-2019
+   Free Software Foundation, Inc.
+
+   The GNU C Library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
```

Comment:
Sorry, we can't use this code :( It's LGPL licensed and we're MIT license, LGPL cannot be included in MIT licensed work.

Reply:
Oh, so sad. Let me find some other implementations.